akka-docs/src/main/paradox/project/migration-guides.md

```diff
@@ -11,8 +11,7 @@
 * [migration-guide-persistence-experimental-2.3.x-2.4.x](migration-guide-persistence-experimental-2.3.x-2.4.x.md)
 * [migration-guide-eventsourced-2.3.x](migration-guide-eventsourced-2.3.x.md)
 * [migration-guide-2.3.x-2.4.x](migration-guide-2.3.x-2.4.x.md)
-* [migration-guide-stream-1.0-2.x](migration-guide-stream-1.0-2.x.md)
```

Comment:
is the reason that the page migration-guide-stream-1.0-2.x.md contains links that are not valid any more?
by removing the page we might introduce a 404 from elsewhere such as some old blog post

Reply:
That was the trigger, yes: we removed `akka-stream-and-http-experimental` entirely so now the links to it 404. I'm not sure there are any links to here (I'd expect them to go directly to `akka-stream-and-http-experimental`) but if so we should fix those as well I think. Perhaps just fix them as they pop up in reports?